__tests__/app.test.js

```diff
@@ -24,6 +24,22 @@ describe("get /api/topics", () => {
     });
 });
 
+describe("get /api", () => {
+    test("returns an object containing a list of available endpoints with descriptions", () => {
+        return request(app)
+            .get("/api")
+            .expect(200)
+            .then(({ body }) => {
+                expect(typeof body.apiDoc === "object" && typeof body.apiDoc !== null).toBeTruthy();
```

Comment:
I'm not the biggest fan of using && inside an expect statement, I would personally do something like 
`expect(typeof body.apiDoc).toBe("object");'
'expect(body.apiDoc).not.toBeNull();`